common/components/org.wso2.carbon.data.agents.log4j/pom.xml

```diff
@@ -61,14 +75,22 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
-                        <!--<Private-Package>org.wso2.carbon.analytics.common.internal.*</Private-Package>-->
-                        <!--<Export-Package>-->
-                            <!--!org.wso2.carbon.analytics.common.internal.*,-->
-                            <!--org.wso2.carbon.analytics.common.spark.*-->
-                        <!--</Export-Package>-->
-                        <!--<Import-Package>-->
-                            <!--org.wso2.carbon.analytics.spark.core.*-->
-                        <!--</Import-Package>-->
+                        <Export-Package>
+                            org.wso2.carbon.data.agents.log4j.*
+                        </Export-Package>
+                        <Import-Package>
+                            org.apache.log4j.*,
+                            org.apache.log4j.spi,
+                            org.wso2.carbon.base.*,
+                            org.wso2.carbon.context.*,
+                            org.wso2.carbon.databridge.agent.*,
+                            org.wso2.carbon.databridge.commons.*,
+                            org.wso2.carbon.logging.service.*,
+                            org.wso2.carbon.user.*,
+                            org.wso2.carbon.utils.*,
```

Comment:
@ruwanta : as a best practice, it is good to define the version range. In WSO2, there are no API changes in minor and patch version, but there can be API changes in major. Therefore we suggest to define the range major to major for wso2 components.